app/services/analytics_events.rb

```diff
@@ -1097,6 +1098,7 @@ def idv_doc_auth_submitted_image_upload_vendor(
       selfie_status:,
       vendor:,
       conversation_id:,
+      reference_id:,
       reference:,
```

Comment:
Hmm I'm not quite sure I understand your question.

My understanding is that this file is mostly for documentation. So including this here is a way of calling it out instead of just passing it through `**extra`. Maybe that's not helpful to your question though.

I am actually surprised it would just be this one event changed though. With it being included on `extra_attributes` it seems like other events might now have this as well.

Reply:
Hm, well the A/C only mentions this one event. Agreed that it's probably available for other events too.

Reply:
Oh sorry I think I wasn't clear - I thought maybe the it was being included in other events on accident. But I just ran through it and didn't see it in other events. So I think it was just me misunderstanding the scope of the code changes. I think only including it on this event is the right choice given the ticket.